README.md

```diff
@@ -1,23 +1,96 @@
-* It is based on an extensively field tested, open architecture with full data, mechanical and electrical redundancy allowing any component to be failed, removed and replaced without the need for a maintenance window. The *ANvil!* platform has over five years of real-world deployment over dozens of sites and an historic uptime of over 99.9999%. 
-* It is extremely easy to use, minimizing the opportunity for human error and making it as simple as possible for "remote hands" to affect repairs and replacements without any prior availability experience and minimal technical knowledge.
+- It is the first server platform designed with the singular focus of protecting your servers and keeping them running under even extreme fault conditions.
+- It is fully self-contained, making it ideal for totally offline operation.
+- It is a "self driving" server availability platform that can continuously monitors internal and external data sources, compiling, analyzing and autonomously deciding when and what action to take to protect your servers. It is ideally suited for extended remote deployments and "hands off" operation.
+- It is based on an extensively field tested, open architecture with full data, mechanical and electrical redundancy allowing any component to be failed, removed and replaced without the need for a maintenance window. The _ANvil!_ platform has over five years of real-world deployment over dozens of sites and an historic uptime of over 99.9999%.
+- It is extremely easy to use, minimizing the opportunity for human error and making it as simple as possible for "remote hands" to affect repairs and replacements without any prior availability experience and minimal technical knowledge.
 
 In short, it is a server platform that just won't die.
 
-* How do you build an *Anvil!*?
+- How do you build an _Anvil!_?
 
 It's quite easy, but it does require a little more space than a README allows for.
-* [How to Build an m2 Anvil!](https://www.alteeve.com/w/Build_an_m2_Anvil!)
 
-The *Anvil!* was designed and extensively tested on [Primergy](http://www.fujitsu.com/global/products/computing/servers/primergy/) servers, Brocade [ICX](http://www.brocade.com/en/products-services/switches/campus-network-switches.html) switches and APC [SmartUPS](http://www.apc.com/smartups/index.cfm?ISOCountryCode=ca) UPSes and [Switched PDU](http://www.apc.com/shop/ca/en/categories/power-distribution/rack-power-distribution/switched-rack-pdu/_/N-17k76am)s. That said, the *Anvil!* platform is designed to be hardware agnostic and should work just fine on Dell, Cisco USC, NEC, Lenovo x-series, and other tier-1 server vendors. 
+- [How to Build an m2 Anvil!](https://www.alteeve.com/w/Build_an_m2_Anvil!)
```

Comment:
_Anvil!_ is in italics everywhere, why not here?

Reply:
Changed as suggested.